pom.xml

```diff
@@ -2169,14 +2118,64 @@
       </build>
     </profile>
 
+    <!-- this must stay as a separate profile as the build plugin has a similar run,
+         but with different dependencies. Maven does not allow 2 similar plugin runs
+         and wrongly combines the 2 which can fail with breaking changes. -->
+    <profile>
+      <id>sevntu</id>
+      <properties>
+        <checkstyle.ant.skip>false</checkstyle.ant.skip>
+      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <executions>
+              <!-- disable non-profile run -->
+              <execution>
+                <id>ant-phase-verify</id>
+                <phase>none</phase>
+              </execution>
+              <!-- profile run -->
+              <execution>
+                <id>ant-phase-verify-sevntu</id>
+                <phase>verify</phase>
+                <goals>
+                  <goal>run</goal>
+                </goals>
+                <configuration>
+                  <target>
+                    <property name="mvn.project.build.directory" value="${project.build.directory}" />
+                    <ant antfile="config/ant-phase-verify-sevntu.xml" />
+                  </target>
+                </configuration>
+              </execution>
+            </executions>
+            <dependencies>
+              <dependency>
+                <groupId>com.puppycrawl.tools</groupId>
+                <artifactId>checkstyle</artifactId>
+                <version>${maven.sevntu-checkstyle-check.checkstyle.version}</version>
```

Comment:
This variable might need rename to skip maven.
As all is done by ant

Reply:
I assumed `maven` was not related to the maven plugin as there are other checkstyle version properties that start with maven also.

maven.sevntu.checkstyle.plugin.version
maven.sevntu-checkstyle-check.checkstyle.version

Should I rename them all and just remove `maven` from the beginning since they are dependency versions and it doesn't really matter how they are run.

Reply:
Is this a change to make in this PR?

Reply:
please read https://github.com/checkstyle/checkstyle/pull/12346/files#r1042257603 , it might result in natural steep to "My recommendations"

Reply:
> My recommendations:

yes, we have small mess in naming 
https://github.com/checkstyle/checkstyle/blob/bef49aea4a923bc003b0ffd1338b8f1606f2badc/pom.xml#L209-L225

I think it is better to remove "maven" from name, we can do this in separate PR/issue, to not complicate current PR diff.

Reply:
Please create issue to not forget to do this

Reply:
Done.